src/css/styles.css

```diff
@@ -0,0 +1,150 @@
+@import url("https://fonts.googleapis.com/css?family=Rubik");
+
+:root {
+  --color-blue: #5776bf;
+  --color-grey: #797a80;
+  --color-light-grey: #f8f8f8;
+  --color-purple: #5a84e8;
+
+  --color-gradient-blue: #61dafb;
+  --color-gradient-purple: #618efb;
+
+  --color-twitter-blue: #55acee;
+  --color-meetup-red: #ed1c40;
+
+  --color-button-text: #3a579b;
+  --color-hyperlink: var(--color-purple);
+  --color-hyperlink-hover: #486aba;
+}
+
+body {
+  font-family: Rubik, Helvetica, Arial, sans-serif;
+  font-size: 20px;
+  line-height: 1.6;
+  color: var(--color-grey);
+  padding: 0;
+  margin: 0;
+}
+
+h1,
+h2,
+h3 {
+  color: var(--color-blue);
+}
+
+h1,
+h2,
+h3,
+.subhead {
+  line-height: 1.2;
+}
+
+h1 {
+  font-size: 56px;
+  font-weight: 300;
+}
+
+h2 {
+  font-size: 44px;
+  font-weight: 300;
+}
+
+h3 {
+  font-size: 20px;
+  font-weight: 300;
+}
+
+.subhead {
+  font-size: 28px;
+}
+
+.ui.button {
+  background-color: white;
+  color: var(--color-button-text);
+}
+
+.ui.button:hover {
+  opacity: 75%;
+}
+
+a {
+  color: var(--color-hyperlink);
+}
+
+a:hover {
+  color: var(--color-hyperlink-hover);
+}
+
+.hero {
+  background-image: url("../assets/hero-bg@2x.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 100vh;
```

Comment:
This might just be personal preference, but I often find it helpful to use `80vh` or `90vh` for the initial section height so that users realize there is more page to scroll down.

![5d080d3860704f000770e692--priceless-meitner-a9781b netlify com_](https://user-images.githubusercontent.com/1153686/59640926-a936af80-912d-11e9-82ed-66aa561a3b69.png)

vs.

![5d080d3860704f000770e692--priceless-meitner-a9781b netlify com_ (1)](https://user-images.githubusercontent.com/1153686/59640936-af2c9080-912d-11e9-9b66-904bd1a451cd.png)

Reply:
@crosscompile I like that too, in some cases, but i left that it took away from the initial image. That's why, on the current design, I added a down chevron to indicate that you ned to scroll to see more.

Reply:
I don't have a strong opinion on this one. It's a simple change so we can go either way and change it later.